modules/alloy/src/main/ts/ephox/alloy/ui/composite/TypeaheadSpec.ts

```diff
@@ -206,6 +214,7 @@ const make: CompositeSketchFactory<TypeaheadDetail, TypeaheadSpec> = (detail, co
         // The navigation here will stop the "previewing" mode, because
         // now the menu will get focus (fake focus, but focus nevertheless)
         navigateList(comp, simulatedEvent, Highlighting.highlightFirst);
+        setAriaActiveDescendant(comp);
```

Comment:
Can we find some way to have highlight set the active descendant automatically? Rather than needing to look up the highlighted item right after we just set it?

Reply:
I found a better place to set active descendant that doesn't require looking up the highlighted item.

Reply:
Looks much better!